terraform/modules/data-share-service/deploy_hook.tf

```diff
@@ -5,43 +5,9 @@ module "deploy_hook" {
   cloudwatch_retention_period = var.cloudwatch_retention_period
   codedeploy_arn              = aws_codedeploy_deployment_group.gdx_data_share_poc.arn
 
-  security_group_id = aws_security_group.lambda.id
-  subnet_ids        = module.vpc.private_subnet_ids
-
-  test_gdx_url  = "http://${aws_lb.load_balancer.dns_name}:8080"
-  auth_url      = module.cognito.token_auth_url
-  client_id     = module.cognito.deploy_hook_client_id
-  client_secret = module.cognito.deploy_hook_client_secret
+  test_gdx_url     = "${local.gdx_api_base_url}:8443"
+  test_auth_header = random_password.test_auth_header.result
+  auth_url         = module.cognito.token_auth_url
+  client_id        = module.cognito.deploy_hook_client_id
+  client_secret    = module.cognito.deploy_hook_client_secret
```

Comment:
Just to check, this change now means it doesn't access our VPC at all, so is back to normal lambda internet access?

Reply:
Exactly